po/glossary/it.po

```diff
@@ -51,7 +51,7 @@ msgstr "numero conto"
 
 #. "the tree view of all accounts"
 msgid "account hierarchy"
-msgstr "struttura conti"
+msgstr "piano dei conti"
```

Comment:
@MocioF At least in german there are 3 different terms:

1. neutral, shown in the CoA: Kontenübersicht oder Kontenhierarchie
2. by authorithies like government or business asssociations: Kontenrahmen
3. by user adjusted form of 2.: Kontenplan

Perhaps we should split it here too? Suggestions?